.github/pull_request_template.md

```diff
@@ -34,6 +34,10 @@ Related Launchpad maas issue in the form `lp#number`.
 
 ## Backports
 
-In general, please propose fixes against *master* rather than release branches (e.g. 2.7), unless the fix is only applicable for that specific release. Please apply backport labels to the PR (e.g. "Backport 2.7") for the appropriate releases to target.
+In general, please propose fixes against _main_ rather than release branches (e.g. 2.7), unless the fix is only applicable for that specific release. Please apply backport labels to the PR (e.g. "Backport 2.7") for the appropriate releases to target.
 
-Only bug and security fixes should be backported, new features should only land in master.
+Only bug and security fixes should be backported, new features should only land in main.
+
+## Screenshots
```

Comment:
Just a driveby I've been meaning to add for ages.

Reply:
Nice. 👍